.docker/nginx-datahub-mock.conf

```diff
@@ -0,0 +1,40 @@
+server {
```

Comment:
A possible followup: if mocks of datahub and biorxiv are useful to other of our repos we could move this to it's own project.